manifests/10_deployment.yaml

```diff
@@ -85,3 +85,28 @@ spec:
             requests:
               cpu: 10m
               memory: 20Mi
+          volumeMounts:
+            - mountPath: /var/run/secrets/serving-cert
+              name: serving-cert
+        - name: vsphere-problem-detector
+          image: registry.svc.ci.openshift.org/ocp/4.7:vsphere-problem-detector
+          terminationMessagePolicy: FallbackToLogsOnError
+          ports:
+          - containerPort: 8444
+            name: vsphere-metrics
+          args:
+          - start
+          - --listen=0.0.0.0:8444
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 10m
+              memory: 20Mi
```

Comment:
btw 20Mi is not going to be enough for this one. I saw it taking 53MB minimum when I was running locally.